app/assets/stylesheets/shareable.scss

```diff
@@ -0,0 +1,10 @@
+nav.share {display:inline-block;} /* shareable's nav tag */
+nav.share div {float:left;}
+nav.share iframe {border:0; margin-top:0; padding-top:0;} /* if using medium sized buttons */
+#___plusone_0 {font-size: default !important;}
+.fb_iframe_widget span { vertical-align: top; }
+.fb_edge_widget_with_comment { vertical-align: top; }
```

Comment:
Selector `fb_edge_widget_with_comment` should be written in lowercase with hyphens